tests/components/zwave_js/test_migrate.py

```diff
@@ -1,534 +1,15 @@
-    assert not set(result["zwave_js_entity_ids"]) ^ set(expected_zwave_js_entities)
-    assert result["migration_entity_map"] == migration_entity_map
-
-    dev_reg = dr.async_get(hass)
-
-    multisensor_device_entry = dev_reg.async_get_device(
-        identifiers={("zwave_js", "3245146787-52")}, connections=set()
-    )
-    assert multisensor_device_entry
-    assert multisensor_device_entry.name_by_user is None
-    assert multisensor_device_entry.area_id is None
-    switch_device_entry = dev_reg.async_get_device(
-        identifiers={("zwave_js", "3245146787-102")}, connections=set()
-    )
-    assert switch_device_entry
-    assert switch_device_entry.name_by_user is None
-    assert switch_device_entry.area_id is None
-
-    migration_device_map = {
-        ZWAVE_SWITCH_DEVICE_ID: switch_device_entry.id,
-        ZWAVE_MULTISENSOR_DEVICE_ID: multisensor_device_entry.id,
-    }
-
-    assert result["migration_device_map"] == migration_device_map
-
-    assert result["migrated"] is False
-
-    ent_reg = er.async_get(hass)
-
-    # no real migration should have been done
-    assert ent_reg.async_is_registered("switch.smart_switch_6")
-    assert ent_reg.async_is_registered("sensor.multisensor_6_battery_level")
-    assert ent_reg.async_is_registered("sensor.smart_switch_6_electric_consumed_w")
-
-    assert ent_reg.async_is_registered(ZWAVE_SOURCE_NODE_ENTITY)
-    source_entry = ent_reg.async_get(ZWAVE_SOURCE_NODE_ENTITY)
-    assert source_entry
-    assert source_entry.unique_id == ZWAVE_SOURCE_NODE_UNIQUE_ID
-
-    assert ent_reg.async_is_registered(ZWAVE_BATTERY_ENTITY)
-    battery_entry = ent_reg.async_get(ZWAVE_BATTERY_ENTITY)
-    assert battery_entry
-    assert battery_entry.unique_id == ZWAVE_BATTERY_UNIQUE_ID
-
-    assert ent_reg.async_is_registered(ZWAVE_POWER_ENTITY)
-    power_entry = ent_reg.async_get(ZWAVE_POWER_ENTITY)
-    assert power_entry
-    assert power_entry.unique_id == ZWAVE_POWER_UNIQUE_ID
-
-    # check that the zwave config entry has not been removed
-    assert hass.config_entries.async_entries("zwave")
-
-    # Check that the zwave integration can be setup after dry run
-    zwave_config_entry = zwave_integration
-    with patch("openzwave.option.ZWaveOption"), patch("openzwave.network.ZWaveNetwork"):
-        assert await hass.config_entries.async_setup(zwave_config_entry.entry_id)
-
-
-async def test_migrate_zwave_not_setup(
-    hass, aeon_smart_switch_6, multisensor_6, integration, hass_ws_client
-):
-    """Test the zwave to zwave_js migration websocket without zwave setup."""
-    entry = integration
-    client = await hass_ws_client(hass)
-
-    await client.send_json(
-        {ID: 5, TYPE: "zwave_js/migrate_zwave", ENTRY_ID: entry.entry_id}
-    )
-    msg = await client.receive_json()
-
-    assert not msg["success"]
-    assert msg["error"]["code"] == "zwave_not_loaded"
-    assert msg["error"]["message"] == "Integration zwave is not loaded"
-
-
-async def test_unique_id_migration_dupes(
```

Comment:
I don't think this test is part of the legacy zwave migration.

Reply:
good catch